charts/redpanda/values.yaml

```diff
@@ -97,6 +97,17 @@ console:
     create: false
   config: {}
 
+#
+# -- Redpanda Managed Connectors settings
+# For a reference of configuration settings,
+# see the [Redpanda Connectors documentation](https://docs.redpanda.com/docs/deploy/deployment-option/cloud/managed-connectors/).
+connectors:
+  enabled: true
+  deployment:
+    create: false
+  test:
```

Comment:
is this to avoid running tests in the redpanda chart? Is this because of line 102 of the connectors/connectors.yaml? If i read correctly you are printing the tests there right?

Reply:
It's only for user convenience. If someone would like to output from helm test of connectors then it is behind this flag.